app/Repositories/TodoRepositories.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Models\Todo;
+use App\Repositories\Interfaces\TodoRepositoryInterface;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Eloquent\Model;
+
+class TodoRepositories implements TodoRepositoryInterface
+{
+    public function index(): Collection|Todo
+    {
+        return Todo::all();
```

Comment:
it's ok when you use method all, but often in a production app we use pagination with limit and offset params. Please, add them to the route